### ISSUE TITLE
Create Dockerfile or Cloud Native Buildpack

### ISSUE THREAD

**Issue by BartlomiejBartnicki** [karunaproject/be#12]
Definition of Done
1. Create buildable solution

Podczas calla został rzuczony pomysł użycia Buildpacka. Prosiłbym o potwierdzenie, że finalnym efektem Puildpacka jest obraz, który możemy podpiąć do docker compose yamla. Jeżeli tak, to nie widzę problemu aby artefaktem budowania BE był własnię taki obraz. Jeżeli się tak nie da to proszę o zastanowienie się czy zostajemy przy dockerze czy może istnieje jakieś inne "ładne" rozwiązanie z buildpackiem.

**Comment by lukasiakp123** [comment]
Tak, powinno sie dać stworzyć obraz używając  Cloud Native Buildpacks i odpalić go za pomocą docker compose. 
Cloud Native Buildpacks dodaje dodatkowe warstwy do obrazu dockera aby zrobic optymalizacje budowania obrazu.

Mimo to uważam że na tym etapie projktu powinismy zostać przy samym dockerze bez komplikowania  deploymentu. 
Jak już bedziemy mieli flow, że aplikacja się buduje na githubie i wrzuca na serwer, możemy pomyśleć o optymalizacjach.
